lib/SyTest/Synapse.pm

```diff
@@ -160,12 +161,17 @@ sub start
 
    my $cert_file = "$self->{hs_dir}/cert.pem";
    my $key_file = "$self->{hs_dir}/key.pem";
+   my $log_config_file = "$self->{hs_dir}/log.config";
+   if( ! -f $log_config_file) {
+      $log_config_file = undef;
+   }
 
    my $macaroon_secret_key = "secret_$self->{port}";
 
    my $config_path = $self->write_yaml_file( config => {
         "server_name" => "localhost:$port",
         "log_file" => "$log",
+        "log_config" => $log_config_file,
```

Comment:
This does mean you'll end up setting the `log_config` key anyway, even if it just gets a YAML undef value. It might be nicer to do

    ( -f $log_config_file ) ? ( log_config => $log_config_file ) : (),

and then also not bother with the `-f` test earlier